codeSnippets/snippets/static-files/src/main/kotlin/com/example/Application.kt

```diff
@@ -1,15 +1,42 @@
 package com.example
 
+import io.ktor.http.*
 import io.ktor.server.application.*
 import io.ktor.server.http.content.*
 import io.ktor.server.routing.*
 import java.io.*
 
 fun Application.module() {
     routing {
+        /*
```

Comment:
why is it commented?

Reply:
Because I am using it only as an example in the topic and wanted to use the same route. It kind of defeats the purpose of having it in code now that I think of it, so I'll comment it out.